src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Completion/RazorCompletionItem.cs

```diff
@@ -33,12 +43,18 @@ internal sealed class RazorCompletionItem : IEquatable<RazorCompletionItem>
             InsertText = insertText;
             Kind = kind;
             CommitCharacters = commitCharacters;
+            SortText = sortText ?? displayText;
```

Comment:
This is the piece that accidentally fixed https://github.com/dotnet/aspnetcore/issues/30255